src/components/business/prevorder.jsx

```diff
@@ -1,9 +1,41 @@
 import React from "react";
+import PrevOrderItem from "./prevorderitem.jsx";
+import axios from "axios";
 
-class PreviousOrder extends React.Component {
+class PreviousOrders extends React.Component {
+  state = {
+    previousOrders: []
+  };
+
+  componentDidMount() {
+    this.getPreviousOrders();
+  }
+
+  getPreviousOrders() {
+    axios.get("http://localhost:7337/api/orders/complete").then(response => {
```

Comment:
You'll want to be sure that you use the environmental variables for this since it will otherwise break in production as you are explicitly always calling `localhost`